examples/zigbee-matter-thermostat/silabs/config/sl_cmp_config.h

```diff
@@ -0,0 +1,74 @@
+/***************************************************************************/ /**
```

Comment:
```suggestion
/*****************************************************************************
```
+ reformat

should probably not be app specific